src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1788,24 +1697,24 @@ namespace HVACUnitaryBypassVAV {
         }
 
         if (CBVAV(CBVAVNum).FanOpModeSchedPtr > 0) {
-            if (GetCurrentScheduleValue(CBVAV(CBVAVNum).FanOpModeSchedPtr) == 0.0) {
-                CBVAV(CBVAVNum).OpMode = CycFanCycCoil;
+            if (ScheduleManager::GetCurrentScheduleValue(CBVAV(CBVAVNum).FanOpModeSchedPtr) == 0.0) {
+                CBVAV(CBVAVNum).OpMode = DataHVACGlobals::CycFanCycCoil;
             } else {
-                CBVAV(CBVAVNum).OpMode = ContFanCycCoil;
+                CBVAV(CBVAVNum).OpMode = DataHVACGlobals::ContFanCycCoil;
             }
         }
 
         // Returns load only for zones requesting cooling (heating). If in deadband, Qzoneload = 0.
-        GetZoneLoads(CBVAVNum, QZnReq);
+        GetZoneLoads(CBVAVNum);
```

Comment:
No longer need (never really needed) QZnReq. This is a temperature based control model so only need to know operating mode, not the actual load.